osbs/api.py

```diff
@@ -286,11 +296,17 @@ def _verify_no_running_builds(self, build_config_name):
             raise OsbsException(msg)
 
     def _create_scratch_build(self, build_request):
+        return self._create_build_directly(build_request)
```

Comment:
shouldn't it be passing unique('scratch') ?

Reply:
Nope. That would allow only a single scratch build at a time in the whole system.
Also, if we do `unique=('git-repo-name', 'git-branch', 'scratch')` it will restrict a single scratch build per repo-branch at a time, which we do not want.
Isolated builds are different, as it should only allow a single build at a time for any given repo-branch.